src/lib/UIKitConfigProvider/index.tsx

```diff
@@ -0,0 +1,121 @@
+import SendbirdChat from '@sendbird/chat';
+import React, { useCallback, useState, createContext, useContext } from 'react';
+
+import { snakeToCamel } from './utils/snakeToCamel';
+import { DeepPartial } from './utils/types';
+
+import { UIKitConfigInfo } from './types';
+
+// @link: https://docs.google.com/spreadsheets/d/1-AozBMHRYOXS74vZ-7x2ptQcBL6nnM-orJWRvUgmkd4/edit#gid=0
+const initialConfig = {
+  common: {
+    enableUsingDefaultUserProfile: false,
+  },
+  groupChannel: {
+    channel: {
+      enableMention: false,
+      enableOgtag: true,
+      enableReactions: true,
+      enableTypingIndicator: true,
+      enableVoiceMessage: false,
+      input: {
+        camera: {
+          enablePhoto: true,
+          enableVideo: true,
+        },
+        enableDocument: true,
+        gallery: {
+          enablePhoto: true,
+          enableVideo: true,
+        },
+      },
+      replyType: 'quote_reply',
+      threadReplySelectType: 'thread',
+    },
+    channelList: {
+      enableMessageReceiptStatus: false,
+      enableTypingIndicator: false,
+    },
+    setting: {
+      enableMessageSearch: false,
+    },
+  },
+  openChannel: {
+    channel: {
+      enableOgtag: true,
+      input: {
+        camera: {
+          enablePhoto: true,
+          enableVideo: true,
+        },
+        enableDocument: true,
+        gallery: {
+          enablePhoto: true,
+          enableVideo: true,
+        },
+      },
+    },
+  },
+};
+
+interface UIKitConfigContextInterface {
+  initDashboardConfigs: (sdk: SendbirdChat) => void;
+  configs: UIKitConfigInfo;
+}
+const UIKitConfigContext = createContext<UIKitConfigContextInterface>({
+  /* eslint-disable @typescript-eslint/no-empty-function */
+  initDashboardConfigs: () => {},
```

Comment:
we can use `noop` instead of /* eslint-disable */
https://github.com/sendbird/sendbird-uikit-react/blob/main/src/utils/utils.js#L1

Reply:
Its type was wrong so fixed the type and also removed the disabling comment. https://github.com/sendbird/sendbird-uikit-react/pull/561/commits/5245991197cc8ef39fd7dfd14086c2d9474679da#diff-7be02629955aad804d99af819265cafc611c0fde160e3eb739e0988c80ac3e4dL66

Reply:
🙆‍♂️